proposed/extended-coding-style-guide.md

```diff
@@ -72,7 +81,7 @@ Code MUST follow all rules outlined in [PSR-1].
 
 All PHP files MUST use the Unix LF (linefeed) line ending.
 
-All PHP files MUST end with a single blank line.
+All PHP files MUST end with a single containing only a single newline (LF) character.
```

Comment:
Looks like you missed a word. Also doesn't look explicit. Should probably be something like

    All PHP files MUST end with a single line, containing only a single newline (LF) character and MUST NOT end with more than 1 consecutive newline (LF) characters.